### PR TITLE
script and some instructions to reproduce the merging stage of the pi…

### PR DESCRIPTION
**Pull request by izcram**
…peline using the original docker images

The docker images required to reproduce the data are now pushed monthly to dockerhub during the release process.
See also https://hub.docker.com/r/brcachallenge/brca-exchange-pipeline

### CODE REVIEW

File: pipeline/docker/reproducibility/reproduce_merging.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+###############################################################################
+# Script to partially reproduce the result from the BRCA Exchange pipeline.
+#
+# Please edit the PLEASE EDIT parts accordingly
+###############################################################################
+
+# Base directory
+# PLEASE EDIT
+BASE=~/please_edit
```
[the remark this answers is not in the document]
Comment (izcram) [reply pipeline/docker/reproducibility/reproduce_merging.sh]: Yes, indeed, it's a bit more canonical. Fixed in d829e31

**Comment by izcram** [comment]
@zfisch What most likely is happening is that file http://brcaexchange.org/backend/downloads/releases/release-02-17-18/release-02-17-18.tar.gz could not be downloaded as we haven't released the data to production yet. It's again one instance of me "anticipating" some URL, sorry about that... Maybe you could retry once the data is ready.

**Comment by zfisch** [comment]
@izcram makes sense! could you add something to make the script exit and display an error message if a release is not available for the date specified?

**Comment by zfisch** [comment]
Still hitting errors with correct dates provided:

```
tar: Pattern matching characters used in file names
tar: Use --wildcards to enable pattern matching, or --no-wildcards to suppress this warning
tar: output/*.vcf*: Not found in archive
tar: Pattern matching characters used in file names
tar: Use --wildcards to enable pattern matching, or --no-wildcards to suppress this warning
tar: output/*.tsv: Not found in archive
tar: Exiting with failure status due to previous errors
```

Also, it would be helpful to include an example of the date format expected and an example command used to run the script, e.g. `reproduce_merging.sh /tmp/reproductions/ 2018-02-17 2018-01-16`

**Pull request merged**